doc/sphinx-guides/source/admin/metadatacustomization.rst

```diff
@@ -481,6 +481,101 @@ These are common ways to use the displayFormat to control how values are display
 |                                   | collection of NMR data.           |
 +-----------------------------------+-----------------------------------+
 
+Metadata Block Setup
+--------------------
+
+Now that you understand the TSV format used for metadata blocks, the next step is to attempt to make improvements to existing metadata blocks or create entirely new metadata blocks. For either task, you should have a Dataverse environment set up for testing where you can drop the database frequently while you make edits to TSV files. Once you have tested your TSV files, you should consider making a pull request to contribute your improvement back to the community.
+
+Exploring Metadata Blocks
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to studying the TSV files themselves you might find the following highly experimental and subject-to-change API endpoints useful to understand the metadata blocks that have already been loaded into your installation of Dataverse:
+
+You can get a dump of metadata fields (yes, the output is odd, please open a issue) like this:
+
+``curl http://localhost:8080/api/admin/datasetfield``
+
+To see details about an individual field such as "title" in the example below:
+
+``curl http://localhost:8080/api/admin/datasetfield/title``
+
+Setting Up a Dev Environment for Testing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You have several options for setting up a dev environment for testing metadata block changes:
+
+- Vagrant: See the :doc:`/developers/tools` section of the Dev Guide.
+- docker-aio: See https://github.com/IQSS/dataverse/tree/develop/conf/docker-aio
+- Full dev environment: See the :doc:`/developers/dev-environment` section of the Dev Guide.
+- AWS deployment: See the :doc:`/developers/deployment` section of the Dev Guide.
+
```

Comment:
In 2f5a1d6 I reordered the list putting "full dev" last.

Reply:
What about the idea of moving it to a more general section like the dev guide? Do you agree / disagree that this could be useful for others than just people dealing with metadata?

(to be clear, we would still leave a note pointing to the new location)

Reply:
@scolapasta meh. I think it's fine where it. If you're hacking on TSV files you need to have some kind of environment set up. Otherwise, how do you test your changes?

Reply:
I agree it's ueful to someone working on TSV files; I'm suggesting it's more broadly useful, so there should be a note hear pointing to this, but that it, for example, could be at the beginning of the dev guide. (since all these are environments that someone developing some aspect could find useful). I just don't want to have us repeat this nice summary of the 4 ways we have to set something up.

@dlmurphy do you have any thoughts on if this should go here or elsewhere?

Reply:
I'd be curious to hear what @jggautier thinks since he's in the TSVs these days. Eleni used to test her changes in Vagrant before pushing changes to GitHub. I helped her get set up.

Reply:
I looked to see if I could find any other place in the guides that lists all four of these dev environment options and I didn't turn anything up. It does seem useful to list these options together, and I think this usefulness extends beyond this page. It might be nice to add a little introduction to the [Development Environment](http://guides.dataverse.org/en/4.9.4/developers/dev-environment.html) section of the Developer Guide that briefly lists all four of these options, provides a sentence or two on when they're useful, and links out to each one's specific section in the guides. This way, one could link to the Development Environment from any other page on the guides.

Reply:
#dlmurphy yes, that it what I was advocating for - putting in the intro of the dev guide.

Reply:
I'm going to be out tomorrow and I'm taking care of some other work right now, so I wouldn't be able to do this until Wednesday at the earliest. So anyone else is welcome to take care of it if they want to, but the idea does have my blessing.